syntax = "proto3";

package ElVis.Serialization;

message LightingModule {

}
